examples/wails/README.md

```diff
@@ -0,0 +1,125 @@
+# GnoNativeKit on Wails
+
+This is a tutorial on how to use GnoNativeKit with Wails. It is based on the official Wails React-TS template.
+
+[Wails](https://wails.app/) it is a framework for building desktop apps by embedding Go and Web runtimes.
+
+## Prerequisites
+- [Wails](https://wails.io/docs/gettingstarted/installation)
+
+## Project Generation
+
+
+### Generate a new Wails React TS project
+
+To generate a new project, run `wails init -n myproject -t react-ts`. This will generate a new project in the current directory.
+Don't worrie, you can check out this [link](https://wails.io/docs/gettingstarted/firstproject#project-layout) and become more familiar with this bunch of generated files.
+
+
+### Configure the GnoNativeKit GRPC Server
+
+We need to change the `app.go` file to start the GnoNative GRPC server:
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+
+	framework "github.com/gnolang/gnomobile/framework/service"
+)
+
+type App struct {
+	ctx    context.Context
+	bridge *framework.Bridge
+}
+
+func NewApp() *App {
+	return &App{}
+}
+
+func (a *App) startup(ctx context.Context) {
+	a.ctx = ctx
+
+	config := framework.NewBridgeConfig()
+	config.UseTcpListener = true
+	config.RootDir = ""
+	bridge, err := framework.NewBridge(config)
+	a.bridge = bridge
+	if err != nil {
+		fmt.Println(err)
+		panic(err)
+	}
+	if bridge != nil {
+		fmt.Println("GnoNative GRPC Server created")
+	} else {
+		fmt.Println("GnoNative GRPC Server not created")
+	}
+}
+```
+
+### Install the Frontend Dependencies
+
+Now we have the GRPC server running, we need to install the GnoNativeKit dependencies in the frontend:
+
+```bash
+cd frontend
+npm install @bufbuild/buf @bufbuild/protobuf @bufbuild/protoc-gen-es @connectrpc/connect @connectrpc/connect-web @connectrpc/protoc-gen-connect-es
+```
+
+Let's copy the protobufs files we already generated for Typescript.
+
+```bash
+cd frontend
+mkdir -p src/api
+cp -r ../../../api/gen/es/* ./src/api/
+cp ../../../templates/es/use-gno-web.ts ./src/api/use-gno.ts
+cp ../../../templates/images/logo-universal.png ./src/assets/images 
+```
+
+Open `src/App.tsx` and add the following code:
```

Comment:
I don't think that "add the following code" should be "replace the contents with the following code"